packages/mds-providers/index.ts

```diff
@@ -93,4 +93,33 @@ const providers: Readonly<{ [id: string]: Readonly<Provider> }> = Object.freeze(
   })
 })
 
-export = providers
+/**
+ * convert provider_id to provider name (if available)
+ * @param  {UUID} provider_id
+ * @return {String} name or provider_id
+ */
+function providerName(provider_id: UUID): string {
+  return providers[provider_id] ? providers[provider_id].provider_name : provider_id
+}
+// /**
```

Comment:
Remove of this commented out code?

Reply:
Yep, I left it in for the review so that people could see the competing versions.  I should have marked the PR as a draft.